packages/manager/apps/pci-rancher/src/components/Modal/UpdateOfferModal/UpdateOfferModal.component.tsx

```diff
@@ -40,17 +41,27 @@ const UpdateOfferModal: FC<UpdateOfferModalProps> = ({
   planInfo,
 }) => {
   const { t } = useTranslation(['dashboard']);
-  const trackAction = useTrackingAction();
-  useTrackingPage(TrackingPageView.UpdateOfferModal);
+  const { trackClick } = useOvhTracking();
 
   const onUpdate = () => {
-    trackAction(TrackingPageView.UpdateOfferModal, TrackingEvent.confirm);
+    trackClick({ actions: [TrackingEvent.confirm] });
+    trackClick({
+      location: PageLocation.popup,
+      buttonType: ButtonType.button,
+      actionType: 'action',
+      actions: [TrackingPageView.UpdateOfferModal, TrackingEvent.confirm],
+    });
```

Comment:
**question:** Is it required to send 2 tracking hits for the same action?

Reply:
You're right, it's a typo.